scripts/merge_tract_cat.py

```diff
@@ -295,9 +295,15 @@ def prefix_columns(cat, filt, fields_to_skip=()):
                         action='store_true', help='Verbose mode.')
     parser.add_argument('--silent', dest='verbose', action='store_false',
                         help='Turn off verbosity.')
+    parser.add_argument('--hsc', dest='hsc', action='store_true',
+                        help='Uses HSC filters')
     args = parser.parse_args(sys.argv[1:])
 
     for tract in args.tract:
         filebase = 'merged_tract_%d' % tract
         filename = filebase+'.hdf5'
-        load_and_save_tract(args.repo, tract, filename, verbose=args.verbose)
+        if args.hsc:
+            load_and_save_tract(args.repo, tract, filename, verbose=args.verbose,
+                                filters=('HSC-U', 'HSC-G', 'HSC-R', 'HSC-I', 'HSC-Z', 'HSC-Y'))
+        else:
+            load_and_save_tract(args.repo, tract, filename, verbose=args.verbose)
```

Comment:
I suggest making this more directly obvious that it's controlling the filters.

```
if args.hsc:
   filters = ('HSC-U', 'HSC-G', 'HSC-R', 'HSC-I', 'HSC-Z', 'HSC-Y')
else:
   filters = ('u', 'g', 'r', 'i', 'z', 'y')

load_and_save_tract(args.repo, tract, filename, verbose=args.verbose, filters=filters)
```

I appreciate that you likely structured it the way you did above to avoid having to make a statement about the filters in the generic case.  But I think it's fine to assume the ugrizy filters as a default at this level.

Eventually we can make this a bit smarter by getting the filter list from the repo.  We can add that logic to this top-level main and then call `load_and_save_tract` with the filter list.  There won't be a need to call anything special options, then.

Reply:
Ok, sure, no problem